backend/src/main/java/ch/nova_omnia/lernello/service/FileService.java

```diff
@@ -0,0 +1,4 @@
+package ch.nova_omnia.lernello.service;
+
+public interface FileService {
```

Comment:
The interface needs the method signatures, and the implementation should then use `@Override`